selfdrive/car/chrysler/interface.py

```diff
@@ -62,13 +62,17 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.wheelbase = 3.88
       ret.steerRatio = 16.3
       ret.mass = 2493. + STD_CARGO_KG
-      ret.minSteerSpeed = 14.5
-      # Older EPS FW allow steer to zero
-      if any(fw.ecu == 'eps' and fw.fwVersion[:4] <= b"6831" for fw in car_fw):
-        ret.minSteerSpeed = 0.
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.minSteerSpeed = 0.5
+      ret.minEnableSpeed = 14.6
+      if car_fw is not None:
+        for fw in car_fw:
+          if fw.ecu == 'eps' and fw.fwVersion in (b"68273275AF", b"68273275AG", b"68312176AE", b"68312176AG", ):
+            ret.minEnableSpeed = 0.
 
     elif candidate == CAR.RAM_HD:
-      ret.steerActuatorDelay = 0.2
+      stiffnessFactor = 0.35
+      ret.steerActuatorDelay = 0.25
```

Comment:
```suggestion
      ret.steerActuatorDelay = 0.2
```
I would prefer to keep this PR for supporting steering down to 0. Let's have tune-specific changes in another PR.

Reply:
Made the changes